### DISCUSSION TITLE
Система автоматизации КПП для ДНП.

### DISCUSSION THREAD

**Discussion by pavel-s-a** [totumonline/totum-ru-issues-and-discussions#574] [Сделанные проекты]
На основе БД Тотум собрали простую но достаточную, удобную и быструю систему автоматического допуска жильцов и гостей на закрытую территорию ДНП. Проведена интеграция с системой видеонаблюдения, контролером Laurent-5G, телеграмм ботом для сбора заявок на пропуска, камерой распознания номеров Dahua, системой ИБП.

Для работы использовали только 3 таблицы, блок Remotes и коны(дорабатываем функции по мере поступления интересных идей ).

![2024-07-08_11-30-34](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/174691756/7e995668-31f6-4b9a-9981-798550fa02c5)
![2024-07-08_11-33-06](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/174691756/88495c98-b2aa-4fbd-8505-e8efd2e7ad6f)
![2024-07-08_11-34-52](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/174691756/3dbe6395-64c2-43c7-a8c7-f8ff9d5d33a7)
Если есть возможность подскажите возможно ли настроить сервер на котором крутится Тотум чтобы все запросы на 80 порт попадали в одну  схему по умолчанию?
 т. к. железяки не умеют DNS имена а людям не особо удобно ходить по IP, да и белый IP не всегда есть возможность получить плюс при его смене система падает(пишет схема не найдена API запросы не принимает) даже локально.

**Comment by LostAlly** [comment]
Nginx нужно настроить чтобы он не передавал имя дальше
`proxy_set_header Host $host;`
а тотум настроить на ip
Только непонятно как тотум урлы генерит, полностью или относительно рута.

**Comment by ttm-alexeypolunin** [comment]
Написал как это сделать здесь: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/575

**Comment by pavel-s-a** [comment]
Всё получилось спасибо огромное.
